src/gui/pages_menu/KM_GUIMenuLobby.pas

```diff
@@ -1718,6 +1739,17 @@ procedure TKMMenuLobby.RefreshSaveList(aJumpToSelected: Boolean);
 end;
 
 
+procedure TKMMenuLobby.MapList_OnShow(Sender: TObject);
+begin
+  if fMapsSortUpdateNeeded then
+  begin
+    //Update sort
+    MapColumnClick(0);
```

Comment:
This will override the users previous sort next time they open the list. For example if I sort by map size, then mark a map as favourite, then close and reopen the list, the sort column will now be favourites not size (and Asc/Desc will flip each time this happens so sometimes favourites will end up on the bottom?). It would be better to trigger the sort again using the previously used sorting mode.